hcl2template/types.build.post-processor.go

```diff
@@ -4,27 +4,43 @@ import (
 	"fmt"
 
 	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/hcl/v2/gohcl"
 	"github.com/hashicorp/packer/packer"
 )
 
 // PostProcessorBlock represents a parsed PostProcessorBlock
 type PostProcessorBlock struct {
 	PType string
-	block *hcl.Block
+	PName string
+
+	HCL2Ref
+}
+
+func (p *PostProcessorBlock) String() string {
+	return fmt.Sprintf(buildPostProcessorLabel+"-block %q %q", p.PType, p.PName)
```

Comment:
This string method will be called by the `%s` option of fmt; it's really handy for ( error ) messages

Reply:
nice.